docs/source/cmd_get.rst

```diff
@@ -6,3 +6,13 @@ onyo get
    :func: setup_parser
    :prog: onyo
    :path: get
+
+
+Example Usage
+*************
+
+**Show all display sizes of the accounting group**
```

Comment:
Similar to above: This is about showing the values of a particular key for all assets underneath a particular path.
Examples that are only making sense in a specific repository should somehow reference such a repository (demo in this case), but point out the general concept of the example call.